xbmc/utils/CMakeLists.txt

```diff
@@ -1,6 +1,7 @@
 set(SOURCES ActorProtocol.cpp
             AlarmClock.cpp
             AliasShortcutUtils.cpp
+            AMLUtils.cpp
```

Comment:
we certainly don't on ios, windows, osx, etc
I am sure our cmake experts can help here. please don't go for the quick and ugly solutions.